### PR TITLE
Fixes issue #6267: Hide Navigation Tabs if Single connected Account

### PR DESCRIPTION
**Pull request by ViditJain123**
I wrapped the TabList with a div and have added conditional display property to make it visible only when we have move than one email address. This fixed the issue. I waiting for the review. :)
Screenshot from 2024-07-16 18-27-37
Screenshot from 2024-07-16 18-27-45

### CODE REVIEW

File: packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsCalendarChannelsContainer.tsx [hunk 1]
```diff
@@ -52,13 +52,16 @@ export const SettingsAccountsCalendarChannelsContainer = () => {
...
   }
+  const hasMultipleAccounts = accounts.length > 1;
 
   return (
     <>
-      <TabList
-        tabListId={SETTINGS_ACCOUNT_CALENDAR_CHANNELS_TAB_LIST_COMPONENT_ID}
-        tabs={tabs}
-      />
+      <div style={{ display: hasMultipleAccounts ? 'block' : 'none' }}>
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsCalendarChannelsContainer.tsx:59]: @ViditJain123 In React we should do something like {hasMultipleAccounts && <TabList ... />}

**Review by charlesBochet — COMMENTED** [review]
Hi @ViditJain123 thanks for the PR, I have left comments :)

**Comment by charlesBochet** [comment]
Also, we should not apply the additional padding in SettingsAccountsMessageChannelDetails in the case we are not in a tab. This results into an additional space. See by comparaison to Appearance page:
image

vs 

image

I think we should extract this padding and place it in a StyledContainer in SettingsAccountsMessageChannelsContainer (resp. CalendarChannelsContainer) instead of having it in SettingsAccountsMessageChannelDetails. This way we could apply it in case of tabs and don't apply it in case of one connected account

File: packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsCalendarChannelsContainer.tsx (hunk 1 above)
Comment (ViditJain123) [reply packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsCalendarChannelsContainer.tsx:59]: > @ViditJain123 In React we should do something like {hasMultipleAccounts && <TabList ... />}

I tried doing that but for some reason, its hiding every component on the screen when handled that way..

**Comment by charlesBochet** [comment]
Actually @ViditJain123, I've just noticed that another PR was opened by @Adithej and that they was assigned on the ticket. I'll prioritize their PR and close this one, sorry about that. Feel free to work on any non-assigned good first issue :)

**Pull request closed without merging**